### PR TITLE
Added Irradiation

### PR DESCRIPTION
**Pull request by baxcoders**
Base: JoulePerSquareMeter (J/m²)
Additional: WattHourPerSquareMeter (Wh/m²) + Prefixes: "Kilo"

### CODE REVIEW

File: UnitsNet/UnitDefinitions/Irradiation.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  "Name": "Irradiation",
+  "BaseUnit": "JoulePerSquareMeter",
+  "XmlDoc": "Irradiation is the process by which an object is exposed to radiation. The exposure can originate from various sources, including natural sources.",
+  "XmlDocRemarks": "https://en.wikipedia.org/wiki/Irradiation",
+  "Units": [
+    {
+      "SingularName": "JoulePerSquareMeter",
+      "PluralName": "JoulesPerSquareMeter",
+      "FromUnitToBaseFunc": "x",
+      "FromBaseToUnitFunc": "x",
```
Comment (angularsen) [inline UnitsNet/UnitDefinitions/Irradiation.json]: Any prefix units to consider?
Any variants of area to consider; squarecentimeter etc ?

File: UnitsNet/UnitDefinitions/Irradiation.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+          "Abbreviations": [ "J/m²" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "WattHourPerSquareMeter",
+      "PluralName": "WattHoursPerSquareMeter",
+      "FromUnitToBaseFunc": "x*3600d",
+      "FromBaseToUnitFunc": "x/3600d",
+      "Prefixes": [ "Kilo" ],
```
Comment (angularsen) [inline UnitsNet/UnitDefinitions/Irradiation.json]: Any variants of area to consider; squarecentimeter etc?

File: UnitsNet/UnitDefinitions/Irradiation.json (hunk 1 above)
Comment (baxcoders) [reply UnitsNet/UnitDefinitions/Irradiation.json]: I'm sorry, we only had occurrence of these specific units, we didn't have any other prefix/variants of that physical quantity in our domain knowledge.

File: UnitsNet/UnitDefinitions/Irradiation.json (hunk 2 above)
Comment (baxcoders) [reply UnitsNet/UnitDefinitions/Irradiation.json]: Same as above.

Comment (angularsen) [reply UnitsNet/UnitDefinitions/Irradiation.json]: That is a perfectly valid reason. We only want to add units that someone actually wants or needs.

**Comment by angularsen** [comment]
The changes are good, but you need to resolve the merge conflict that ocurred due to merging in some other PRs just now.

**Comment by angularsen** [comment]
It's probably just a matter of regenerating the code once more.

**Comment by angularsen** [comment]
Sigh.. to be *more* clear, it is probably simply a matter merging in latest `origin/master`, and either fix the merge conflicts manually or run `regenerate-code.bat` and mark the conflicted files as fixed.

**Comment by ferittuncer** [comment]
I merged `angularsen/master` into `baxcoders:Module_10` and resolved merge conflicts. It should be okay now.

**Pull request merged**